addon/styles/partials/_uni-footer-website.scss

```diff
@@ -0,0 +1,13 @@
+.uni-footer-website {
+    @include footer-website();
+
+    &__row {
+        &__cell {
+            &__anchor {
+                &--inline {
```

Comment:
being this just a single selector nesting and being this an exceptional case I would collapse into a single selector.